Formula/r/rink.rb

```diff
@@ -0,0 +1,35 @@
+class Rink < Formula
+  desc "Unit conversion tool and library written in rust"
+  homepage "https://rinkcalc.app/about"
+  url "https://github.com/tiffany352/rink-rs/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "40048e84c2b606e50bf05dec2813acedeb48066cd48537d0dea453a72d000d60"
+  license all_of: ["MPL-2.0", "GPL-3.0-or-later"]
+
+  depends_on "asciidoctor" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "zlib"
```

Comment:
```suggestion
```

Reply:
The macOS build doesn't link against system `zlib` on macOS, so we can remove this.

```
╰─ brew linkage rink
System libraries:
  /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
  /usr/lib/libSystem.B.dylib
  /usr/lib/libcurl.4.dylib
  /usr/lib/libiconv.2.dylib
```